style/base.css

```diff
@@ -3,3 +3,13 @@
 
     https://jupyterlab.readthedocs.io/en/stable/developer/css.html
 */
+
+.jp-ToolbarButtonComponent[data-command='jhub-apps:deploy-app'] svg {
```

Comment:
Can you use child combinator here? Bare selector on right hand side can impact performance of the entire app. Browser match s lectors right to left traversing entire dom tree (optimizations like bloom filters aside). Adding a more specific rule allows browser to short circuit after looking just at the parent.

Reply:
Thanks Mike, I didn't appreciate this, really great advice! 🙏